runtimes/pythonrt/runner/tee.py

```diff
@@ -0,0 +1,51 @@
+from queue import Queue
+from threading import Thread
+from time import sleep
+import grpc
+import pb
+
+
+class Tee:
+    def __init__(self, stream, runner_id, worker):
+        self.stream = stream
+        self.queue = Queue()
+        self.running = True
+        self.worker = worker
+        self.runner_id = runner_id
+
+        Thread(target=self.print_thread, daemon=True).start()
+
+    def close(self):
+        self.running = False
+        self.stream.close()
+
+    def write(self, data):
+        self.stream.write(data)
+        self.queue.put(data)
+
+    def flush(self):
+        self.stream.flush()
+        while not self.queue.empty():
```

Comment:
This might not finish if the thread doing gRPC calls is stuck. Limit this by time as well.